labelbox/client.py

```diff
@@ -899,6 +907,30 @@ def create_ontology_from_feature_schemas(self,
         normalized = {'tools': tools, 'classifications': classifications}
         return self.create_ontology(name, normalized, media_type)
 
+    def delete_unused_feature_schema(self, feature_schema_id):
+        """
+        Deletes a feature schema if it is not used by any ontologies or annotations
+
+        Args:
+            feature_schema_id (str): The id of the feature schema to delete
+
+        Returns:
+            True if the feature schema was deleted
+        """
+
+        endpoint = self.rest_endpoint + "/feature-schemas/" + feature_schema_id
```

Comment:
- Could you URL encode this endpoint?
- Could you also validate `feature_schema_id` to be a string of length 25? (cuid)